apps/tilbakemelding-api/src/main/resources/application.yml

```diff
@@ -0,0 +1,20 @@
+AAD_ISSUER_URI: https://login.microsoftonline.com/62366534-1ec3-4962-8869-9b5535279d0b
+CLIENT_ID: 6554cc46-d5a9-4b9b-aafb-775b9ee09696
+
+spring:
+  application:
+    version: @application.version@
+    name: tilbakemelding-api
+    description: App for innsendinger tilbakemeldinger. Tilbakemeldingene blir publisert i slack
```

Comment:
App for innsending av tilbakemeldinger. Tilbakemeldingene blir publisert i slack